### PR TITLE
Adding a template for Azure Application Gateway test setup

### PR DESCRIPTION
**Pull request by AlekseiPolkovnikov**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
*
*
*
*
*

### Description of the change

### CODE REVIEW

**Comment by AlekseiPolkovnikov** [comment]
@bmoore-msft  could you please help to resolve: travis is timing out (https://travis-ci.org/Azure/azure-quickstart-templates/builds/159708627) however this App GW setup/startup in the template that is making the whole thing run about ~30min (as expected). Any possibility to overcome this timeout issue?

File: application-gateway-demo-setup/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+    "subnetPrefix": "10.0.0.0/28",
+    "webServerTemplateLocation": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/ubuntu-apache-test-page/",
+    "webServerTemplateName": "azuredeploy.json",
+    "server1TestPage": "<h1 style=\"color:red;font-size:300%;\">This is Server 1</h1>",
+    "server2TestPage": "<h1 style=\"color:blue;font-size:300%;\">This is Server 2</h1>",
+    "serverTestPageInfo": "<p>Send next request. If Cookie-based affinity is enabled, clear the cookies to change the backend server.</p><p><strong>Request headers:</strong> <br /><?php $hs = apache_request_headers();foreach($hs as $h => $value){echo \"$h: $value <br />\n\";}?></p>"
+  },
+  "resources": [
+    {
+      "apiVersion": "[variables('apiVersionSubDeployment')]",
```
Comment (bmoore-msft) [inline application-gateway-demo-setup/azuredeploy.json]: use a literal value for apiVersion - so schema validation can be done.

File: application-gateway-demo-setup/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+            "value": "Server 1"
+          },
+          "installPHP": {
+            "value": true
+          }
+        }
+      }
+    },
+    {
+      "apiVersion": "[variables('apiVersionSubDeployment')]",
```
Comment (bmoore-msft) [inline application-gateway-demo-setup/azuredeploy.json]: use literal apiVersion value

File: application-gateway-demo-setup/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+            "value": "Server 2"
+          },
+          "installPHP": {
+            "value": true
+          }
+        }
+      }
+    },
+    {
+      "apiVersion": "[variables('apiVersion')]",
```
Comment (bmoore-msft) [inline application-gateway-demo-setup/azuredeploy.json]: use literal apiVersion value

File: application-gateway-demo-setup/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+      "type": "Microsoft.Network/publicIPAddresses",
+      "name": "[variables('publicIPAddressName')]",
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "publicIPAllocationMethod": "Dynamic"
+      }
+    },
+    {
+      "apiVersion": "[variables('apiVersion')]",
```
Comment (bmoore-msft) [inline application-gateway-demo-setup/azuredeploy.json]: use literal apiVersion value

File: application-gateway-demo-setup/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+            "name": "[variables('subnetName')]",
+            "properties": {
+              "addressPrefix": "[variables('subnetPrefix')]"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "apiVersion": "[variables('apiVersion')]",
```
Comment (bmoore-msft) [inline application-gateway-demo-setup/azuredeploy.json]: use literal apiVersion value

**Comment by AlekseiPolkovnikov** [comment]
@bmoore-msft thanks, done

**Comment by AlekseiPolkovnikov** [comment]
@bmoore-msft good to go or anything else to fix?

**Comment by bmoore-msft** [comment]
yep - extra button now...

**Pull request merged**